phpBB/phpbb/ucp/controller/webpush.php

```diff
@@ -135,7 +135,6 @@ public function notification(): JsonResponse
 
 		// Decode and return data if everything is fine
 		$data = json_decode($notification_data, true);
-		$data['url'] = isset($data['url']) ? $this->path_helper->update_web_root_path($data['url']) : '';
```

Comment:
Still need to figure out if this is / was needed.

Reply:
I'm just glad I wasn't who came up with this line so I can't be directly blamed for it's existence 😂

Reply:
By the way, if we don't need that line, there's no need to decode the JSON to an array. We can just send the `$notification_data` in the `JsonResponse` using the `$json` argument:
```php
return new JsonResponse($notification_data(), 200, [], true);
```